modules/inbucket/inbucket_test.go

```diff
@@ -0,0 +1,71 @@
+package inbucket
+
+import (
+	"context"
+	"net/smtp"
+	"testing"
+
+	"github.com/inbucket/inbucket/pkg/rest/client"
+	"github.com/testcontainers/testcontainers-go"
+)
+
+func TestInbucket(t *testing.T) {
+	ctx := context.Background()
+
+	container, err := RunContainer(ctx, testcontainers.WithImage("inbucket/inbucket:sha-2d409bb"))
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	// Clean up the container after the test is complete
+	t.Cleanup(func() {
+		if err := container.Terminate(ctx); err != nil {
+			t.Fatalf("failed to terminate container: %s", err)
+		}
+	})
+
+	// smtpConnection {
+	smtpUrl, err := container.Smtp(ctx)
+	// }
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	// webInterface {
+	webInterfaceUrl, err := container.WebInterface(ctx)
+	// }
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	restClient, err := client.New(webInterfaceUrl)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	headers, err := restClient.ListMailbox("to@example.org")
+	if err != nil {
+		t.Fatal(err)
+	}
+	if len(headers) != 0 {
+		t.Fatalf("expected 0 messages, got %d", len(headers))
+	}
+
+	msg := []byte("To: to@example.org\r\n" +
+		"Subject: testcontainers test!\r\n" +
+		"\r\n" +
+		"This is a testcontainer test.\r\n")
```

Comment:
Proper naming is important 😄 
```suggestion
		"This is a Testcontainers test.\r\n")
```

Reply:
Done